content/docs/2.4/concepts/scaling-jobs.md

```diff
@@ -136,7 +140,9 @@ Select a Scaling Strategy. Possible values are `default`, `custom`, or `accurate
 >		runningJobs++
 >}
 >```
->`PendingJobCount` provides an indication of the amount of jobs that are in pending state, for example a that has not finished yet **and** the underlying pod is either not running or has not completed yet.
+>`PendingJobCount` provides an indication of the amount of jobs that are in pending state. Pending jobs can be calculated in two ways:
+> - Default behavior - Job that has not finished yet **and** the underlying pod is either not running or has not completed yet (Default Behavior)
```

Comment:
```suggestion
> - Default behavior - Job that have not finished yet **and** the underlying pod is either not running or has not been completed yet
```

Reply:
👍 
I've added your suggestion + fixed the code block (that has changed in my linked PR): d76801d